src/components/TrungDay3CRUD/PageLocal/localread.js

```diff
@@ -0,0 +1,157 @@
+import { useState } from "react";
+import { Table, Button, Checkbox, Form } from "semantic-ui-react";
+
+const data = [
+  {
+    firstName: "Trung",
+    lastName: "Chi Nguyen",
+    checkbox: true,
+    id: 1,
+  },
+  {
+    firstName: "Toan",
+    lastName: "Nguyen",
+    checkbox: true,
+    id: 2,
+  },
+  {
+    firstName: "React",
+    lastName: "Crud",
+    checkbox: true,
+    id: 3,
+  },
+  {
+    firstName: "Nhan",
+    lastName: "Nguyen",
+    checkbox: true,
+    id: 4,
+  },
+  {
+    firstName: "Gandaf",
+    lastName: "iajsdf",
+    checkbox: true,
+    id: 5,
+  },
+];
+
+export default function ReadLocal() {
+  const [localData, setLocalData] = useState(data);
+  const [firstName, setFirstName] = useState("");
```

Comment:
common lại thành object không define nhiều state

Reply:
Solved! Pls check it out.